docs/docusaurus/docs/reference/learn/glossary.md

```diff
@@ -3,6 +3,26 @@ id: glossary
 title: "Glossary"
 ---
 
-import InProgress from '../../core/_core_components/_in_progress.md'
+[Action](/core/validate_data/checkpoints/manage_checkpoints.md): A Python class with a `run` method that takes a Validation Result and does something based on its contents.
 
-<InProgress/>
+[Batch](/core/manage_and_access_data/manage_batches.md): A selection of records retrieved from a Data Asset according to the parameters passed to a Batch Definition.
```

Comment:
Shouldn't these be pointing to the glossary pages? Or are you thinking that we're just going down to the 1 sentence versions exclusively?

Reply:
Since the existing glossary "learn" content was falling out of date, and a fair amount of the new 1.0 objects didn't have existing "learn" content or further changed how the existing objects interacted I left out the original learn topics and linked to the practicals for this draft.

I think that revising old and creating new learn topics for each of these subjects will probably mean sufficient lift to be put into its own epic as a concerted effort, and until then I'd prefer that we link to the up-to-date practical guides than risk overlooking content that should be revised by patching in the old docs and inadvertently sending people to outdated information.